src/startscene.js

```diff
@@ -115,73 +94,38 @@ function populatePlanetarySystem(scene) {
         color: new Color3(0, 0.3, 1),
         rocky: false
     };
-    planets.push(createPlanet(hg, scene));
-    planets.push(createPlanet(aphro, scene));
-    planets.push(createPlanet(tellus, scene));
-    planets.push(createPlanet(ares, scene));
-    planets.push(createPlanet(zeus, scene));
-    return planets;
-}
-
-function createSpinAnimation() {
-    let orbitAnim = new Animation("planetspin", "rotation.y", 30, Animation.ANIMATIONTYPE_FLOAT, Animation.ANIMATIONLOOPMODE_CYCLE);
-    const keyFrames = [];
-    keyFrames.push({
-        frame: 0,
-        value: 0
-    });
+    const planetData = [ hg, aphro, tellus, ares, zeus ];
+    const planets = [];
+    planetData.forEach(p => {
+        const planet = AstroFactory.createPlanet(p, scene);
+        AstroFactory.createAndStartOrbitAnimation(planet, scene);
+        planet.computeWorldMatrix(true);
+        let planetTrail = new TrailMesh(planet.name + "-trail", planet, scene, .1, planet.orbitOptions.orbitalCircum, true);
+        let trailMat = new StandardMaterial(planetTrail.name + "-mat", scene);
+        trailMat.emissiveColor = trailMat.specularColor = trailMat.diffuseColor = planet.orbitOptions.color;
```

Comment:
same question as before - is that intentional?

Reply:
yes, but only due to lack of imagination! If you have any loose heuristics that you use when slapping down values like this I'm all ears